docs/faststore/docs/customization/building-sections/override-use-cases/adding-more-icon-options-to-the-alert-component.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: "Adding more icon options to the Alert component"
+---
+
+This guide illustrates a common use case for [`getOverriddenSection`](https://developers.vtex.com/docs/guides/faststore/overrides-getoverriddensection-function) in FastStore. It focuses on customizing the Headless CMS schema for a section to provide more icon options.
+
+> ℹ️ For detailed instructions on overriding components, refer to [Overriding native component’s props](https://developers.vtex.com/docs/guides/faststore/overrides-component-props) and [Overriding a native component](https://developers.vtex.com/docs/guides/faststore/overrides-native-component).
+
+## Context
+
+- The native [Alert](https://developers.vtex.com/docs/guides/faststore/building-sections-list-of-native-sections#alert) section provides the following icon options by default:
```

Comment:
@Mariana-Caetano I think here its better to show the hCMS interface to illustrate:

https://github.com/vtexdocs/dev-portal-content/assets/3356699/a86df9ad-360d-4ee6-9725-955a01578197